setup-app.py

```diff
@@ -24,3 +77,96 @@
 
 # Now the app can be started
 call(["cf", "start", appname])
+
+# Tail the application log
+print("Parsing log files.")
+end_token = "cf stop %s" % appname  # Seeing this in the log means certs done
+log_pipe = Popen("cf logs %s --recent" % appname, shell=True,
+                 stdout=PIPE, stderr=PIPE)
+log_lines = log_pipe.stdout.readlines()
+print("Waiting for certs (could take several minutes)")
+while end_token not in ''.join(log_lines):
```

Comment:
infinite loop if something goes wrong while retrying the certs.  should probably have some sort of cap on this.
maybe either check the logs for failure OR tell the user to do so